src/cpp/benders/factories/BendersFactory.cpp

```diff
@@ -35,7 +34,11 @@ int RunBenders(char** argv, const std::filesystem::path& options_file,
         std::filesystem::path(options.OUTPUTROOT) / "reportbenders.txt";
 
     Writer writer;
-
+    const auto couplig_map = build_input(benders_options.STRUCTURE_FILE);
+    const auto method = (options.BATCH_SIZE == 0 ||
```

Comment:
Can you extract in a function with if for readability and debugging purpose please

Reply:
And also typo `couplig_map` -> `coupling_map`